File: shivam-chaurasiya/Practice Problems/Level 1/01 Beginner Programs/Problem18.py
# Average of Marks in 5 Subjects

math = int(input('Enter the marks of math: '))
physics = int(input('Enter the marks of physics: '))
chemistry = int(input('Enter the marks of chemistry: '))
Hindi = int(input('Enter the marks of Hindi: '))
English = int(input('Enter the marks of English: '))

total_marks = math+physics+chemistry+Hindi+English
average_marks = (math+physics+chemistry+Hindi+English)/5

print(f'Total marks is: {total_marks} and average marks is: {average_marks}')

File: Basic Programs/Problem_18.js
/*--------------------------------
Problem 18: Average of Marks in 5 Subjects

Write a program that takes marks obtained in 5 subjects as input, calculates the total marks and average
marks, and displays both.
--------------------------------*/

const maths = 37;
const physics = 43;
const english = 64;
const hindi = 51;
const chemistry = 43;

const average = maths + physics + english + hindi + chemistry / 5;
console.log(`Average is: ${average}`);

/* OUTPUT:
Average is: 203.6
*/
